core/src/account_provider.rs

```diff
@@ -124,8 +124,8 @@ impl AccountProvider {
         Ok((address, public))
     }
 
-    pub fn remove_account(&self, address: Address, password: &Password) -> Result<(), SignError> {
-        Ok(self.keystore.write().remove_account_with_password(&address, password)?)
```

Comment:
It seems that `remove_account_with_password` is not used anymore.

Reply:
I think it can be deleted.

Reply:
But multistore can have the same account with different password. Is this intended? `remove_account` will delete all of them.

Reply:
I found out that `KeyStore` blocks the multi-account situation on `KeyMultistore`. I think it's okay to delete it.